flocker/testtools/skips.py

```diff
@@ -0,0 +1,75 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Utilities to help with skipping tests.
+"""
+from os import environ
+
+
+def fail_if_skipped(test_item):
+    """
+    Check whether to fail the given test object if it will be skipped.
+
+    :param test_object: test method or class with a skip decorator.
+
+    :return: True if ``test_object`` is covered by environment variable
+        FLOCKER_TEST_FAIL_IF_SKIPS, else False.
+    """
+    test_fail_if_skips = environ.get("FLOCKER_TEST_FAIL_IF_SKIPS")
+    return True
```

Comment:
This can be changed to see the effect, when running `trial flocker.sample` of changing the `FLOCKER_TEST_FAIL_IF_SKIPS` environment variable once this is implemented.

The design is that running:
`FLOCKER_TEST_FAIL_IF_SKIPS=flocker.sample trial flocker.sample` would cause all the skipping tests in that module to fail, while running `FLOCKER_TEST_FAIL_IF_SKIPS=flocker.another_module trial flocker.sample` would cause no tests to fail.